erpnext/manufacturing/report/bom_stock_calculated/bom_stock_calculated.py

```diff
@@ -15,24 +15,39 @@ def execute(filters=None):
 
 	for row in data:
 		item_map = get_item_details(row.item_code)
+		manufacture_details = get_manufacturer_records(row.item_code)
```

Comment:
it should be called outside for loop. Even `get_item_details` should be called outside.

Reply:
done